### PR TITLE
Reduce log spam in dockertools manager.go

### LINKED ISSUE #26972: "Generating ref for container" log message is spammy
For example:

```
manager.go:1508] Generating ref for container POD: &api.ObjectReference{Kind:"Pod", Namespace:"e2e-tests-pods-ybngf", Name:"pod-hostip-a37443a4-2c9e-11e6-8f55-0242ac110007", UID:"a38da480-2c9e-11e6-b631-42010af00026", APIVersion:"v1", ResourceVersion:"307", FieldPath:"implicitly required container POD"}
```

Could we consider moving it to log level 4 or 5?

cc @kubernetes/sig-node @smarterclayton @kubernetes/rh-cluster-infra

### PR DESCRIPTION
**Pull request by pmorie**
Fixes #26972

@kubernetes/sig-node

### CODE REVIEW

File: pkg/kubelet/dockertools/manager.go [hunk 1]
```diff
@@ -1503,9 +1503,9 @@ func (dm *DockerManager) runContainerInPod(pod *api.Pod, container *api.Containe
 
 	ref, err := kubecontainer.GenerateContainerRef(pod, container)
 	if err != nil {
-		glog.Errorf("Can't make a ref to pod %v, container %v: '%v'", pod.Name, container.Name, err)
+		glog.Errorf("can't make a ref to pod %v, container %v: '%v'", pod.Name, container.Name, err)
 	}
-	glog.Infof("Generating ref for container %s: %#v", container.Name, ref)
+	glog.V(5).Infof("generated ref for container %s: %#v", container.Name, ref)
```
Comment (dims) [inline pkg/kubelet/dockertools/manager.go]: Do we want to print this inside an else {} block? (as in the failure case, we'll see both lines 1506 and 1508 in the logs

File: pkg/kubelet/dockertools/manager.go [hunk 2]
```diff
@@ -1552,7 +1552,7 @@ func (dm *DockerManager) runContainerInPod(pod *api.Pod, container *api.Containe
 	containerLogFile := containerInfo.LogPath
 	symlinkFile := LogSymlink(dm.containerLogsDir, kubecontainer.GetPodFullName(pod), container.Name, id.ID)
 	if err = dm.os.Symlink(containerLogFile, symlinkFile); err != nil {
-		glog.Errorf("Failed to create symbolic link to the log file of pod %q container %q: %v", format.Pod(pod), container.Name, err)
+		glog.Errorf("failed to create symbolic link to the log file of pod %q container %q: %v", format.Pod(pod), container.Name, err)
```
Comment (dims) [inline pkg/kubelet/dockertools/manager.go]: Unrelated change?

Comment (pmorie) [inline pkg/kubelet/dockertools/manager.go]: It's unrelated, but log messages are supposed to begin with lowercase.  Fixed it while I was in there.

File: pkg/kubelet/dockertools/manager.go (hunk 1 above)
Comment (pmorie) [inline pkg/kubelet/dockertools/manager.go]: Accepted.

File: pkg/kubelet/dockertools/manager.go (hunk 2 above)
Comment (dims) [inline pkg/kubelet/dockertools/manager.go]: Ack thanks! just getting a feel of what's normal in kubernetes community as i am learning

-- Dims

**Comment by ncdc** [comment]
LGTM

File: pkg/kubelet/dockertools/manager.go (hunk 2 above)
Comment (yujuhong) [inline pkg/kubelet/dockertools/manager.go]: @pmorie is there a style guide for lowercase log messages? I've seen both in the k8s codebase, and advocates for both sides as well. Maybe I've missed the latest trend?

Comment (ncdc) [inline pkg/kubelet/dockertools/manager.go]: I checked the various .md files in the repo and couldn't find anything. I know that error messages must start with a lowercase letter.

**Comment by pmorie** [comment]
@k8s-bot test this issue: #26431

**Comment by ncdc** [comment]
@k8s-bot node e2e test this issue: #26431

File: pkg/kubelet/dockertools/manager.go (hunk 2 above)
Comment (timstclair) [inline pkg/kubelet/dockertools/manager.go]: Error's are addressed [here](https://github.com/golang/go/wiki/CodeReviewComments#error-strings), but not log messages. I believe the convention is for log messages to start with a capital letter.
```console
$ ag --noheading --nobreak 'glog\.[A-Za-z]+\("[A-Z]' pkg | wc -l
869
$ ag --noheading --nobreak 'glog\.[A-Za-z]+\("[a-z]' pkg | wc -l
231
```
